client/src/components/Navbar.jsx

```diff
@@ -1,27 +1,43 @@
-import React from "react";
-import { useHistory } from "react-router-dom";
+import {Navbar as BootstrapNavbar, Nav} from 'react-bootstrap/'
```

Comment:
`import {Navbar as BootstrapNavbar, Nav} from 'react-bootstrap/'`, you can drop the `/` here